atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -67,6 +67,9 @@ def run(self):
         """
         Pull parent images and retag them uniquely for this build.
         """
+        if self.workflow.builder.custom_base_image or self.workflow.builder.base_from_scratch:
+            return
```

Comment:
Info about skipping plugin here as well?

Reply:
we don't want to skip for base_from_scratch, there can still be some pullable parents

Reply:
Fixed.
Not sure how to test it, though.

Reply:
you can just test that when you set workflow.builder.custom_base_image
you will get that log message, and maybe that it won't run set_base_image

Reply:
Simple test case added.